src/common/models/dialog.ts

```diff
@@ -56,6 +57,9 @@ export interface DialogType {
         feed: IOpdsFeedView;
     };
     [DialogTypeName.ApiappAddForm]: {};
+    [DialogTypeName.DeletePublicationConfirm]: {
```

Comment:
same why re-added the previous modal definition for deletePublicationConfirm

Reply:
Same reason, error from my part. I copied some things from the other branch and those shouldn't have been.